packages/engine-server/src/backup/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./service";
```

Comment:
No real reason it _has_ to be in `engine-server`. `filesv2` is in `common-server` and backup is pretty adjacent to files so that could be one argument for moving it there.

all service classes we implement are in `engine-server` so I followed that convention here.